contract/external.go

```diff
@@ -0,0 +1,60 @@
+/**
+* This a temp file that mock the external APIs used in smart contract component.
+* It will be removed when blockchain infrastructure constructed, including account,
+* security, RPC, consensus and blockchain storage components.
+*
+* @file
+* @copyright defined in go-seele/LICENSE
+ */
+package contract
+
+import (
+	"github.com/seeleteam/go-seele/crypto"
+)
+
+const (
+	txTypeContractReg int = iota
+	txTypeContractCall
+	txTypeContractDel
+)
+
+// Account wraps the information of smart contract account.
+type Account struct {
+	AccountAddress string
+	CodeAddress    string
+	State          []byte
```

Comment:
add some comments for these members.

Reply:
I suppose State's type is something like enum

Reply:
All the code in external.go are temp, so please ignore the comment.

Reply:
The state should be of type object, but use []byte for serialization.